tests/main/lxd-mount-units/task.yaml

```diff
@@ -51,9 +55,14 @@ execute: |
     retry --wait 1 -n 10 sh -c 'lxd.lxc exec ubuntu -- systemctl --wait is-system-running | grep -Eq "(running|degraded)"'
 
     lxd.lxc file push --quiet "$GOHOME"/snapd_*.deb "ubuntu/root/"
-
     DEB=$(basename "$GOHOME"/snapd_*.deb)
+    
     lxd.lxc exec ubuntu -- apt update
+    if os.query is-ubuntu 22.10; then
```

Comment:
Please add a comment explaining what we are doing here (and why it's needed for 22.10 only).

Reply:
added

Reply:
thanks for the review